src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cardLogoWidth: 160px;
+$cardLogoHeight: 134px;
+$cardWidth: 166px;
+$lineHeitghtBig: 18px;
+$lineHeitghtMiddle: 16px;
+$lineHeitghtSmall: 14px;
+$fontSizeUltra: 16px;
+$fontSizeBig: 14px;
+$fontSizeMiddle: 12px;
+$fontSizeSmall: 10px;
+$addishionalCardColor: #616070;
```

Comment:
The variable `$addishionalCardColor` seems to have a typo. It should be `$additionalCardColor` to correct the spelling of 'additional'.